### ISSUE TITLE
linespace after opening curly

### ISSUE THREAD

**Issue by mpadge** [ropensci-review-tools/spaceout#1]
Currently does this:
``` r
code <- "
f <- function () {

return (1)
}"
styler::style_text (code, style = spaceout::spaceout_style)
#> 
#> f <- function () {
#>     return (1)
#> }
```

<sup>Created on 2021-10-04 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

Need to retain that first white line.

### Reason

White lines are arbitrarily retained at any and all arbitrary places within the code, as should well be, and yet they are systematically removed from that one place only - when occurring immediately after an opening curly (and also immediately before closing). These automated interventions, tidyverse-conform as they may be, are inconsistent. White lines are always an arbitrary imposition, and as such should be left. Ping @maelle

**Comment by maelle** [comment]
Related https://github.com/r-lib/styler/issues/371

**Comment by maelle** [comment]
What's worrying to me is that I do not understand which rule currently removes the empty line.

**Comment by lorenzwalthert** [comment]
we named it like this because around meant before and after curly braces for us.

**Comment by lorenzwalthert** [comment]
That's for all rules that have `around`.

**Comment by maelle** [comment]
Yeah I was thinking "curly braceS" so `{ bla }`, not curly braces on their own. Definitely wrong thinking on my side!!
